tests/routes/userRoutes.spec.ts

```diff
@@ -0,0 +1,177 @@
+import { Response, Request } from 'express';
+import bcrypt from 'bcryptjs';
+import AuthUser from '../../src/middleware/authUser';
+import UserController from '../../src/controllers/userController';
+import User from '../../src/models/userModel';
+
+const userController = new UserController();
+const userMock = {
+  _id: '3472417428',
+  email: 'natan@gmail.com',
+  password: '123',
+  phone: '56565777',
+  name: 'Jerson',
+  state: 'Goias',
+  city: 'Rio Verde',
+  admin: true,
+};
+const mockResponse = () => {
+  const response = {} as Response;
+  response.status = jest.fn().mockReturnValue(response);
+  response.sendStatus = jest.fn().mockReturnValue(response);
+  response.json = jest.fn().mockReturnValue(response);
+  return response;
+};
+
+describe('Test Create User function', () => {
+  it('should get a statusCode 200', async () => {
```

Comment:
Deixar o teste mais descritivo, por exemplo
"Should get a statusCode 200 when create a user with the right data"